pom.xml

```diff
@@ -1,51 +1,147 @@
-				<artifactId>spring-boot-maven-plugin</artifactId>
-				<configuration>
-					<classifier>exec</classifier>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
-	<distributionManagement>
-		<repository>
-			<id>spring-cloud-stream-modules</id>
-			<name>spring-cloud-stream-modules</name>
-			<url>http://repo.spring.io/spring-cloud-stream-modules</url>
-		</repository>
-		<snapshotRepository>
-			<id>spring-cloud-stream-modules-snapshot</id>
-			<name>spring-cloud-stream-modules snapshot</name>
-			<url>http://repo.spring.io/spring-cloud-stream-modules</url>
-		</snapshotRepository>
-	</distributionManagement>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-cloud-stream-modules</artifactId>
+    <groupId>org.springframework.cloud.stream.module</groupId>
+    <packaging>pom</packaging>
+    <url>http://projects.spring.io/spring-xd/</url>
+    <version>1.0.0.BUILD-SNAPSHOT</version>
+    <organization>
+        <name>Pivotal Software, Inc.</name>
+        <url>http://www.spring.io</url>
+    </organization>
+    <properties>
+        <spring-cloud.version>Brixton.BUILD-SNAPSHOT</spring-cloud.version>
+        <spring-framework.version>4.2.0.RC2</spring-framework.version>
+        <spring-integration.version>4.2.0.M2</spring-integration.version>
+    </properties>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.3.0.BUILD-SNAPSHOT</version>
+    </parent>
+
+
+    <dependencyManagement>
+
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream-codec</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream-common</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream-binder-local</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.cloud</groupId>
+                <artifactId>spring-cloud-stream-binder-redis</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
```

Comment:
We shouldn't include both the Redis and Rabbit binder implementations: with both on the CP, there will be a conflict (we need to solve this in spring-cloud-stream)